content/scripts/report-coverage.sh

```diff
@@ -5,4 +5,6 @@ set -o pipefail -eu
 
 PATH="${PWD}/bin:${PATH}"
 
+pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+
```

Comment:
There are two issues here:

1) Not every branch will be using devel. To avoid having to modify this script for each collection, we'll need to pass this in as a variable from the pipeline.
2) This isn't necessary for ansible/ansible, so to keep the script the same for both collection and non-collection use cases the installation here should be conditional on the variable noted above.